metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "simp-auditd",
-  "version": "8.1.0",
+  "version": "8.1.1",
```

Comment:
You don't actually have to bump the version for test changes or static asset changes, as those are not mission-impacting changes.  Did you bump the version because you want to release this?

Reply:
@lnemsick-simp Yes, I bumped it to prep it for the next release.